Docs/source/acknowledge_us.rst

```diff
@@ -39,12 +39,84 @@ Please add the following sentence to your publications, it helps contributors ke
 Latest WarpX reference
 **********************
 
-If your project leads to a scientific publication, please consider citing the paper below.
+If your project leads to a scientific publication, please consider citing all co-authors via our persisten Zenodo DOI:
+
+.. tab-set::
+
+   .. tab-item:: LateX: Bibtex
+
+      .. code-block:: bibtex
+
+      @misc{WarpX,
+  author       = {Vay, Jean-Luc and
+                  Almgren, Ann and
+                  Amorim, Lígia Diana and
+                  Andriyash, Igor and
+                  Angus, Justin Ray and
+                  Belkin, Daniel and
+                  Bizzozero, David and
+                  Blelly, Aurore and
+                  Clark, Stephen Eric and
+                  Fedeli, Luca and
+                  Formenti, Arianna and
+                  Garten, Marco and
+                  Ge, Lixin and
+                  Gott, Kevin and
+                  Harrison, Cyrus and
+                  Huebl, Axel and
+                  Giacomel, Lorenzo and
+                  Groenewald, Roelof E. and
+                  Grote, David and
+                  Gu, Junmin and
+                  Haseeb, Muhammad and
+                  Jambunathan, Revathi and
+                  Klion, Hannah and
+                  Kumar, Prabhat and
+                  Thévenet, Maxence and
+                  Richardson, Glenn and
+                  Shapoval, Olga and
+                  Lehe, Remi and
+                  Loring, Burlen and
+                  Miller, Phil and
+                  Myers, Andrew and
+                  Rheaume, Elisa and
+                  Rowan, Michael E. and
+                  Sandberg, Ryan Thor and
+                  Scherpelz, Peter and
+                  Weichman, Kale and
+                  Yang, Eloise and
+                  Zaim, Neïl and
+                  Zhang, Weiqun and
+                  Zhao, Yinjian and
+                  Zhu, Kevin Z. and
+                  Zoni, Edoardo},
+          title        = {WarpX: an advanced electromagnetic & electrostatic Particle-In-Cell code},
+          year         = 2018,
+          publisher    = {Zenodo},
+          version      = {25.08},
+          doi          = {10.5281/zenodo.16750069},
+          url          = {https://doi.org/10.5281/zenodo.16750069},
+          swhid        = {swh:1:dir:12d584a99e9625391d4bc82fbbca1a089652f3f8
+                           ;origin=https://doi.org/10.5281/zenodo.4571577;vis
+                           it=swh:1:snp:42a98189c61704a75545d176f00552bd43071
+                           d45;anchor=swh:1:rel:5843c56c1006a8673ba5ea31e0fc8
+                           0fd36afcf45;path=BLAST-WarpX-warpx-6680268
+                          },
```

Comment:
Do we need this for the citation? Isn't the URL and DOI enough to uniquely identify this project?
```suggestion
```